src/__tests__/lib/containers/SynapseTable.test.tsx

```diff
@@ -1,309 +1,266 @@
-        "WHERE species='Human' " +
-        "AND assay='rnaSeq'"
-      const columnIndexes: number[] =
-        instance.getCountFunctionColumnIndexes(originalSql)
-      expect(columnIndexes).toHaveLength(0)
-    })
-    it('group by with count function', async () => {
-      const { instance } = createShallowComponent(props)
-      const originalSql: string =
-        'SELECT bar, baz, count(distinct id) as files, concat(biz)' +
-        'FROM syn987654321 ' +
-        "WHERE species='Human' " +
-        "AND assay='rnaSeq' group by 1, 2"
-      const columnIndexes: number[] =
-        instance.getCountFunctionColumnIndexes(originalSql)
-      expect(columnIndexes).toEqual([2])
-    })
-    it('group by without count function', async () => {
-      const { instance } = createShallowComponent(props)
-      const originalSql: string =
-        'SELECT bar, baz, concat(distinct id) as files, concat(biz)' +
-        'FROM syn987654321 ' +
-        "WHERE species='Human' " +
-        "AND assay='rnaSeq' group by 1, 2"
-      const columnIndexes: number[] =
-        instance.getCountFunctionColumnIndexes(originalSql)
-      expect(columnIndexes).toHaveLength(0)
-    })
+    )
+    expect(screen.queryByTestId('HasAccess')).not.toBeInTheDocument()
   })
 
   describe('create table headers works', () => {
-    it('renders correctly', () => {
-      const { wrapper } = createShallowComponent(props)
+    it('renders correctly', async () => {
+      renderTable(props, queryContext)
       // there are a total of 13 columns in view, so we expect
       // 13 headers
-      expect(wrapper.find('th')).toHaveLength(totalColumns)
+      expect(await screen.findAllByRole('columnheader')).toHaveLength(
+        totalColumns,
+      )
       // there are five facets for the dataset so there should be 5
       // faceted columns
-      expect(wrapper.find(EnumFacetFilter)).toHaveLength(5)
-      expect(wrapper.find('th.SRC-hidden')).toHaveLength(8)
+      expect(
+        await screen.findAllByRole('button', {
+          name: 'Filter by specific facet',
+        }),
+      ).toHaveLength(5)
     })
 
     it('handle column sort press works', async () => {
       /*
-        Overview:
-          Go through clicking a column's sort button, there are
-          three states that cycle:
-            - off
-            - descending
-            - ascending
-      */
-      const { wrapper, instance } = createShallowComponent(props)
+          Overview:
+            Go through clicking a column's sort button, there are
+            three states that cycle:
+              - off
+              - descending
+              - ascending
+        */
+      renderTable(props, queryContext)
       // simulate having clicked the sort button on the first column
-      // projectName -- this should set it to descend
-      const sortedColumn = 'projectName'
+      // projectStatus -- this should set it to descend
+      const sortedColumn = 'projectStatus'
```

Comment:
The item that comes first changed, and I'm not sure why. Unusual, but I don't think a regression occurred.